model/commitqueue/github_pr_sender.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const CommitQueueContext = "evergreen/commitqueue"
```

Comment:
Nit: This should just be called "Context":

From https://blog.golang.org/package-names:

> Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.